src/dagmc.cpp

```diff
@@ -609,14 +630,39 @@ void DAGUniverse::uwuw_assign_material(
   fatal_error("DAGMC was not configured with UWUW.");
 #endif // OPENMC_UWUW
 }
+
+void DAGUniverse::override_assign_material(std::string key,
+  moab::EntityHandle vol_handle, std::unique_ptr<DAGCell>& c) const
+{
+
+  // if Cell ID matches an override key, use it to override the material
+  // assignment else if UWUW is used, get the material assignment from the DAGMC
+  // metadata
+  std::stringstream msg;
+  msg << "XML instance"
+      << material_overrides.count("id_" + std::to_string(c->id_));
+  write_message(msg.str(), 5);
```

Comment:
I made use of `fmt` here as we do in other output messages and changed the verbosity level to 8. I don't think we want to see all of this information by default.

Can you incorporate this and update the content of the output message a bit. What information is meant to be provided to the user here?

```suggestion
  write_message(fmt::format("XML instance {}", c->id_), 8);
```